solr/core/src/java/org/apache/solr/api/AnnotatedApi.java

```diff
@@ -222,7 +223,8 @@ public void call(SolrQueryRequest req, SolrQueryResponse rsp) {
     final String command;
     final MethodHandle method;
     final Object obj;
-    ObjectMapper mapper = SolrJacksonAnnotationInspector.createObjectMapper();
+    ObjectMapper mapper = SolrJacksonAnnotationInspector.createObjectMapper()
```

Comment:
`ObjectMapper` is a relatively heavy object, we should not create new instances in every class that needs it - maybe put a common instance somewhere in Utils?